Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+# Cheatsheet
+## Commonly used functions on [Array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays)
+### [1. map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)
+
+**Description:**
+Creates an array by calling a specific function on each element present in the parent array.
+
+**Syntax:**
+* _array.map(function(currentValue, index, arr), thisValue)_
+
+| Parameter | Description |
+| :-------- | :---------- |
+| function(currentValue,index,arr) | Required function to be run for each array element |
+| thisValue | Optional. |
+
+**Return value:** 
+An Array containing the results of calling the provided function for each element in the original array.
+
+### [2. filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)
+
+**Description:**
+Creates an array filled with all array elements that pass a test (provided as a function).
+
+**Syntax:**
+ * _array.filter(function(currentValue, index, arr), thisValue)_
+
+| Parameter | Description |
+| :-------- | :---------- |
+| function(currentValue,index,arr) | Required function to be run for each array element |
+| thisValue | Optional. |
+
+**Return value:**
+Array containing all elements that pass the test else an empty array.
+
+**Example:**
+```
+let filtered = [112, 52, 0, -1, 944].filter(value => value>0);

```

Comment:
please add spaces like `value > 0`

and for **all example code** in this doc, please use `function` functions, not `=>` arrow functions since this doc is for beginners

Reply:
Done.